packages/cli/src/commands/serve/serve.ts

```diff
@@ -70,7 +70,7 @@ export async function serveMesh({ baseDir, argsPort, getBuiltMesh, logger, rawCo
   const protocol = sslCredentials ? 'https' : 'http';
   const serverUrl = `${protocol}://${hostname}:${port}`;
   if (!cluster.isWorker && fork) {
```

Comment:
This should be `fork != null`, right?

Reply:
This condition allows to test both `fork !== false || fork !== null`, right?

Reply:
It will also accept the values even if they are falsy but defined.

Reply:
Better? https://github.com/Urigo/graphql-mesh/pull/3479/commits/75263a5c181e68881d7f1ea7f03558249d11ea2d

Reply:
@ardatan 🆙 👀